todotxtio.py

```diff
@@ -1,5 +1,9 @@
+#!/usr/bin/python
```

Comment:
hmmm... I always create my librarys so that in the end, they can be used as libraries AND can be executed as standalone tools (where this makes sense). So, you are right, here the line makes no sense, yet.